src/main/java/com/hierynomus/msfscc/fileinformation/FileStreamInformation.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (C)2016 - SMBJ Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hierynomus.msfscc.fileinformation;
+
+import java.util.ArrayList;
+
+
+public class FileStreamInformation implements FileQueryableInformation {
+
+    private ArrayList <FileStreamInformationItem> streamList;
```

Comment:
Could you declare this as `List` instead of `ArrayList`. Better to limit the scope of this implementation choice.